packages/bot-web-ui/src/components/blockly-loading/blockly-loading.tsx

```diff
@@ -1,22 +1,20 @@
 import React from 'react';
 import { Loading } from '@deriv/components';
-import { connect } from 'Stores/connect';
-import RootStore from 'Stores/index';
+import { observer } from '@deriv/stores';
+import { useDBotStore } from '../../stores/useDBotStore';
 
-type TBlocklyLoadingProps = {
-    is_loading: boolean;
-};
+const BlocklyLoading = observer(() => {
+    const { blockly_store } = useDBotStore();
```

Comment:
```suggestion
    const { blockly_store } = useStore();
```

`blockly_store` is coming from `RootStore` right ?

Reply:
Then it's fine